podvm/Makefile

```diff
@@ -35,7 +35,6 @@ endif
 	$(eval OPTS := -var qemu_image_name=${IMAGE_FILE} \
 	-var cloud_image_url=${RHEL_IMAGE_URL} \
 	-var disk_size=11144 \
-	-var ssh_username=cloud-user \
```

Comment:
Isn't the default for a RHEL cloud image cloud-user ? If you remove this, what username does packer login to the booted vm as ?

Reply:
AFAIU we set it here: https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/podvm/qcow2/userdata.cfg

Reply:
That's the cloud-init config which configures the booted up vm. But how does the SSH communicator in packer know how to connect to the vm as if you don't specify ssh_username ?

Reply:
It's specified here - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/podvm/qcow2/rhel/qemu-rhel.pkr.hcl#L14

Reply:
but also here:
 [ qemuargs          = [["-m", "${var.memory}"], ["-smp", "cpus=${var.cpus}"], ["-cdrom", "${var.cloud_init_image}"], ["-serial", "mon:stdio"], ["-cpu", "Cascadelake-Server"]]](https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/podvm/qcow2/rhel/qemu-rhel.pkr.hcl#L11)
 
which one will have the priority ?

Reply:
`qemuargs` are not related to the `ssh` userid/password.

Reply:
Ah, I didn't realize it's also specified in the main file. I was really confused how packer didn't complain about ssh_username missing :)

Reply:
@bpradipt it has the cloud-init-image there, which have the ssh_user in it no?

Reply:
@snir911 yes.